hooks/environment

```diff
@@ -14,151 +14,68 @@ set -euo pipefail # exit on failure or unset variable
 source /etc/profile.d/modules.sh
 
 export SLURM_BUILDKITE_PATH="$(dirname "${BUILDKITE_BIN_PATH}")"
+export GIT_SSH_COMMAND="ssh -i ${SLURM_BUILDKITE_PATH}/.ssh/id_rsa -o IdentitiesOnly=yes"
 export SHARED_DEPOT="${BUILDKITE_BUILD_PATH}/shared_depot"
-export GIT_SSH_COMMAND="ssh -i ${SLURM_BUILDKITE_PATH}/.ssh/id_rsa -o IdentitiesOnly=yes -o UserKnownHostsFile=${SLURM_BUILDKITE_PATH}/.ssh/known_hosts"
 export CI_BUILD_DIR="${BUILDKITE_BUILD_PATH}/${BUILDKITE_PIPELINE_SLUG}/${BUILDKITE_BUILD_NUMBER}"
+export JULIA_DEPOT_PATH="${JULIA_DEPOT_PATH:-$CI_BUILD_DIR/depot/default}:${SHARED_DEPOT}"
 export BUILDKITE_BUILD_CHECKOUT_PATH="${CI_BUILD_DIR}/${BUILDKITE_PIPELINE_SLUG}"
 export BUILDKITE_API_TOKEN=$(cat "${SLURM_BUILDKITE_PATH}/.buildkite_token")
 
-export TMPDIR="/tmp/slurm-${SLURM_JOB_ID}"
-srun --ntasks-per-node=1 --ntasks=${SLURM_JOB_NUM_NODES} mkdir -p "${TMPDIR}"
-printenv TMPDIR
-
-ulimit -c unlimited
-
 # Slurm Job Info
 echo "--- Slurm Job ID: ${SLURM_JOB_ID}"
 scontrol show job ${SLURM_JOB_ID} --details
 scontrol show node ${SLURM_JOB_NODELIST}
 
-if $(grep -q "Red Hat" /etc/redhat-release);  then
-    echo "We are on a RedHat 9 node"
-
-    # We are on rhel9 nodes, we also want our modules
-    export MODULEPATH="/groups/esm/modules:$MODULEPATH"
-
-    module load git/2.39.3-gcc-11.3.1-zfr3sti
-else
-    # We are on the CentOS nodes
-    module load git/2.26.0
-fi
-
-
-# this is set to avoid rare race conditions on the same node with concurrent (step) jobs
-export OMPI_MCA_orte_tmpdir_base="$TMPDIR"
-
-case "${BUILDKITE_AGENT_META_DATA_CONFIG}" in
-    # standard configurations
-   ""|default)
-        module load ${BUILDKITE_AGENT_META_DATA_MODULES:-}
-        export JULIA_DEPOT_PATH="${JULIA_DEPOT_PATH:-$CI_BUILD_DIR/depot/default}:${SHARED_DEPOT}"
-        ;;
-
-    init)
-        echo "--- init env configuration"
-        ;;
+export TMPDIR="/tmp/slurm-${SLURM_JOB_ID}"
 
-    cpu)
-        echo "--- cpu env configuration"
-        module load singularity/3.5.2
-        module load python3/${PYTHON_VERSION:=3.8.5}
-        module load julia/${JULIA_VERSION:=1.10.0}
-        if [ "${MPI_IMPL:=mpich}" == "mpich" ]; then
-            module load mpich/${MPICH_VERSION:=4.0.0} hdf5/1.12.1 netcdf-c/4.8.1
+case "$BUILDKITE_AGENT_META_DATA_QUEUE" in
+    "new-central")
+        echo "Queue: new-central"
+        echo "SLURM_NODELIST: $SLURM_NODELIST"
+    
+    # Try SSH connection
+        if ! ssh -o ConnectTimeout=10 -o BatchMode=yes $SLURM_NODELIST "echo Test connection successful"; then
```

Comment:
`$SLURM_NODELIST` is in general a number of nodes, does this work with more than one node?